src/drive/mobile/ducks/mediaBackup/index.js

```diff
@@ -86,32 +67,39 @@ export const startMediaBackup = (dirName, force = false) => async (
         ) {
           break
         }
-        dispatch(currentUploading(photo, uploadCounter++, totalUpload))
-        await dispatch(uploadPhoto(dirName, dirID, photo))
+        dispatch(currentMediaUpload(photo, uploadCounter++, totalUpload))
+        await dispatch(uploadPhoto(targetFolderName, dirID, photo))
       }
     }
   }
 
   cozy.client.settings.updateLastSync()
-  dispatch(endMediaUpload())
+  dispatch({ type: MEDIA_UPLOAD_END })
 }
 
+const mediaUploadSucceed = ({ id }) => ({
+  type: MEDIA_UPLOAD_SUCCESS,
+  id
+})
+
 const uploadPhoto = (dirName, dirID, photo) => async (dispatch, getState) => {
   try {
     await cozy.client.files.statByPath(dirName + '/' + photo.fileName)
-    dispatch(successMediaUpload(photo))
+    dispatch(mediaUploadSucceed(photo))
     return
   } catch (_) {} // if an exception is throw, the file doesn't exist yet and we can safely upload it
 
   try {
-    const maxBackupTime = 1000 * 60 * 5 // 5 minutes
+    const MILLISECOND = 1
+    const SECOND = 1000 * MILLISECOND
+    const MINUTE = 60 * SECOND
+    const maxBackupTime = 5 * MINUTE
```

Comment:
🤣

Reply:
With good variable's names, I can remove comments.
I think this code is way more expressive than 
```js
const maxBackupTime = 1000 * 60 * 5 // 5 minutes
```
Maybe there is no need to be as explicit as it is, maybe a simple code like this is enough:
```js
const FIVEMINUTES = 300000
const maxBackupTime = FIVEMINUTES
```
But I want to show that a good naming worth any comments.

Reply:
Yes I understood that :). I just think in this instance it's taking things too far, as

```
const maxBackupTime = 1000 * 60 * 5 // 5 minutes
```

…is more readable to me than

```
const MILLISECOND = 1
const SECOND = 1000 * MILLISECOND
const MINUTE = 60 * SECOND
const maxBackupTime = 5 * MINUTE
```

…but I don't mind it :)

Reply:
I can't agree more, it is way more readable.
But, what about:
```js
const maxBackupTime = 1000 * 60 * 60 * 5 // 5 minutes
```
or
```js
const maxBackupTime = 1000 * 60 * 15 / 2 // 5 minutes
```
or
```js
const maxBackupTime = 600 * 5 // 5 minutes
```

Such numbers are called *magic number*. Why magic? Because it comes from nowhere and need a comment to explain its existence. But comments are pledges you can't believe with closed eyes. If not, then you need time to understand what means this magic number and how the comment helps or not.

So I think it is less readable at the end of the day, because you can't trust what you read.

Reply:
Entre les étiquettes qu'on trouve sur les boîtes dans une cuisine (riz, pâtes, lentilles, etc.), les étiquettes qu'on trouve sur les sonnettes des immeubles, voire même la pancarte "peinture fraîche" sur un banc public. Je confirme : les commentaires sont des promesses.